app/src/main/res/values/strings.xml

```diff
@@ -29,4 +29,5 @@
     <string name ="settings">settings</string>
     <string name ="choose_level_dialog_text">Select your level, please. You can change it later
         in settings.</string>
+    <string name="move_cursor_to_solved_squares_text">move cursor to solved squares</string>
```

Comment:
It's not quite clear IMO. I would say "type/print solved squares" or "not skip filled squares"